conda-forge.yml

```diff
@@ -1,9 +1,12 @@
 build_platform:
   osx_arm64: osx_64
+conda_build:
+  pkg_format: '2'
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

Comment:
you could try `linux_ppc64le: azure` here: it will be _even slower_, but _might_ not timeout.

Reply:
(and then ask for a re-render)

Reply:
Ok, let‘s try this.

Reply:
`Successful in 153m` seems long vs the next-longest-build... but better than travis "oh-no-its-monday" or whatever causes things to just not happen there.

Reply:
The build is now passing, however the pcc64le builds take 2.5 hours.  This doesn't look not very practical.  Shouldn't we disable that architecture altogether?